src/main/java/dgen/column/ColumnGenerator.java

```diff
@@ -47,10 +48,10 @@ public ColumnGenerator(ColumnConfig columnConfig) {
             this.ang = new RandomAttributeNameGenerator(rnd);
         }
 
-        if (columnConfig.getObject("datatype") == null) {
+        if (columnConfig.getObject("column.type") == SpecType.DEFFOREIGNKEY) {
```

Comment:
It does not mean 'data type' based on this line, so probably good to explain what that 'type' above means.

Reply:
Fixed in 4ec4ff75482108855ea85906f3e59ab4525f0b98